understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-                     Whether it is remembering random strings of characters, or a pattern gesture to perform
-                     on a touch screen, cognitive function tests will exclude some people. When a cognitive
-                     function test is used, at least one other authentication method must be available
-                     which is not a cognitive function test.
+                  <p>サイト固有のパスワードを記憶することは、<a href="#dfn-cognitive-function-test">認知機能テスト</a>である。そのようなテストは、多くの認知障害のある人々にとって問題となることが知られている。ランダムな文字列を記憶する場合でも、タッチスクリーン上で実行するパターンジェスチャを記憶する場合でも、認知機能テストは一部の人を排除する。認知機能テストを使用する場合、認知機能テストではない他の認証方法が少なくとも一つ使用可能でなければならない。
                   </p>
                   
                   
                   
-                  <p>Some <abbr title="Completely Automated Public Turing test to tell Computers and Humans Apart">CAPTCHA</abbr> systems have an audio alternative of the visible text. If the user needs to transcribe
-                     this audio, it cannot be used to meet the Alternative exception.
+                  <p>一部の <abbr title="Completely Automated Public Turing test to tell Computers and Humans Apart">CAPTCHA</abbr> システムには、可視テキストの音声代替がある。利用者がこの音声を転記する必要がある場合、代替の例外を満たすために使用することはできない。
                   </p>            
                   
                   
-                  <p>If there is more than one step in the authentication process, such as with multi-factor
-                     authentication, all steps need to comply with this Success Criterion to pass. There
-                     needs to be a path through authentication that does not rely on cognitive function
-                     tests.
+                  <p>多要素認証のように、認証プロセスに複数のステップがある場合、合格するには全てのステップがこの達成基準に適合する必要がある。認知機能テストに依存しない認証のパスが必要である。
                   </p>
                   
                   
-                  <p>Being able to recover or change the email and password is an important part of authentication.
-                     If the user is authenticating with alternative information in order to recover their
-                     account, there needs to be a method that is not a cognitive function test.
+                  <p>電子メール及びパスワードを回復又は変更できることは、認証の重要な部分である。利用者がアカウントを回復するために代替情報で認証している場合、認知機能テストではない方法が必要である。
                   </p>
                   
                   
-                  <p>Many organizations are required to use 2-factor authentication that combines independent
-                     sources to confirm a user's identity. These sources can consist of combining authentication
-                     through:
+                  <p>多くの組織は、利用者の ID 身元を確認するために、独立したソースを組み合わせた 2 要素認証を使用することが要求される。これらのソースは、次の認証を組み合わせて構成できる:
                   </p>
                   
                   
                   <ul>
                      
-                     <li>knowledge (e.g., password, letters in a passphrase or memorized swipe path);</li>
+                     <li>知識 (例えば、パスワード、パスフレーズ内の文字、又は記憶されたスワイプパス)。</li>
```

Comment:
swipe path の path は、「軌跡ベースのジェスチャ (path-based gestures)」の path と同義かと思いますので、以下、いかがでしょうか？

```suggestion
                     <li>知識 (例えば、パスワード、パスフレーズ内の文字、又は記憶されたスワイプの軌跡)。</li>
```